packages/truffle-box/lib/utils/unbox.js

```diff
@@ -81,13 +76,45 @@ function fetchRepository(url, dir) {
   });
 }
 
-function copyTempIntoDestination(tmpDir, destination) {
-  return new Promise(function(accept, reject) {
-    fs.copy(tmpDir, destination, function(err) {
-      if (err) return reject(err);
-      accept();
-    });
-  });
+async function copyTempIntoDestination(tmpDir, destination) {
+    const currentContent = fs.readdirSync(destination);
+      if (currentContent.length > 1) {
```

Comment:
All this code inside this if statement is very difficult to read for me.  I wonder if we need all the try catch statements, is it possible to use a catch statement on the outside of this method?  And is there any reason to use the async fs methods as opposed to the synchronous ones?  We could get rid of all the awaits then.

Reply:
I made `copyTemp...` into an async func so I could have `inquirer` pause the processes while waiting for the user to respond. I'll take a look at this again though to see If using callbacks or promises can help clear things up.

Reply:
Oh sorry, it wasn't the fact that you made it async.  I was trying to think of a way to simplify the indentation mountain so that the code narrative was more clear.